vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -47,9 +47,15 @@ namespace Microsoft.ReactNative
     void ReplaceChild(XAML_NAMESPACE.FrameworkElement parent, XAML_NAMESPACE.UIElement oldChild, XAML_NAMESPACE.UIElement newChild);
   }
 
-  [webhosthidden]
+  [webhosthidden] DOC_STRING("Enables a view manager to be responsible for its own layout and sizing.")
   interface IViewManagerRequiresNativeLayout
   {
     Boolean RequiresNativeLayout { get; };
   }
+
+  [webhosthidden]
+  DOC_STRING("Enables a view manager to dynamically decide what kind of view to create depending on the properties passed to it.")
+  interface IViewManagerCreateWithProperties {
+    Object CreateViewWithProperties(IJSValueReader propertyMapReader);
```

Comment:
Why is this just an object? Shouldn't it be FrameworkElement to be more in line with the existing CreateView? Or at least a DependencyObject (if we don't want to change the existing CreateView).

Reply:
long term, we want to enable modules to be dual-compiled for winui and system xaml. By having the API be dialect agnostic we can set ourselves up for this long term, as opposed to be tied to a UI-framework-specific type